api/metadata/annotations.go

```diff
@@ -15,21 +15,21 @@
 package metadata
 
 const (
-	domain                      = "sw.kogito.kie.org"
-	Key                         = domain + "/key"
-	Name                        = domain + "/name"
-	Description                 = domain + "/description"
-	Annotations                 = domain + "/annotations"
-	DataInputSchema             = domain + "/dataInputSchema"
-	ExpressionLang              = domain + "/expressionLang"
-	Metadata                    = domain + "/metadata"
-	Version                     = domain + "/version"
-	Label                       = domain + "/label"
-	Profile                     = domain + "/profile"
-	SecondaryPlatformAnnotation = domain + "/secondary.platform"
-	OperatorIDAnnotation        = domain + "/operator.id"
+	Domain                      = "sw.kogito.kie.org"
+	Key                         = Domain + "/key"
```

Comment:
as these are constants, maybe make all uppercase?

Reply:
Go convention does not mandate const to be uppercase since the public/private definitions are taken from lower/upper case of the member name.